test/functionalTest/cases/947_v2_entities/v2_entities_simple.test

```diff
@@ -0,0 +1,426 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Each issue will have it's own directory, of course.
We've been working like that for over a year ...
But that's not really the point here, we're talking about the name of the file, not the directory.
Perhaps we should include the issue number in the name of the files as well?

Right now the test file is named GET_v2_entities.test.

Just let me know if it needs a change.

Detail:

    % testHarness.sh GET_v2_entities.test

Here the directory name is not taken into account. If we have more than one test file named GET_v2_entities.test then both/all will be executed.
If we include the issue number in the test-file, this will never happen